kokoro/presubmit/presubmit.sh

```diff
@@ -85,12 +86,41 @@ function run_buildozer() {
 }
 
 function run_enumerate_tests() {
-  TARGETS="$($BAZEL query --deleted_packages=//gapii/fuchsia --output label 'kind(".*_test rule", //...)' | sort -t: -k1,1 | awk '{print "        \""$0"\","}')"
+  # Get all the test targets in all packages except gapii/fuschia
+  TARGETS=$($BAZEL query --deleted_packages=//gapii/fuchsia --output label 'kind(".*_test rule", //...)')
+
+  # Exclude linting targets from tests
+  TARGETS=$(echo "$TARGETS" | grep -v ":lint_")
+
+  # Sort the test targets alphabetically
+  TARGETS=$(echo "$TARGETS" | sort -t: -k1,1)
+
+  # Indent the test target names to match the BUILD.bazel file
+  TARGETS=$(echo "$TARGETS" | awk '{print "        \""$0"\","}')
+
   OUT=$(mktemp)
   cp BUILD.bazel $OUT
   cat $OUT | awk -v "targets=${TARGETS//$'\n'/\\n}" 'begin {a=0} /__END_TESTS/ {a=0} { if (a==0) print $0;} /__BEGIN_TESTS/ { a=1; print targets }' > BUILD.bazel
 }
 
+function run_enumerate_lints() {
+  # Get all the linting test targets in Vulkan generator
+  TARGETS=$($BAZEL query 'attr(generator_function, py_lint, //vulkan_generator/...)')
+
+  # Exclude the sub targets to prevent repetation
+  TARGETS=$(echo "$TARGETS" | grep -v "_flake8" | grep -v "_mypy" | grep -v "_pylint")
+
+  # Sort the linting targets alphabetically
+  TARGETS=$(echo "$TARGETS" | sort -t: -k1,1)
```

Comment:
Can we factor this common stuff out somehow?

Reply:
I was thought of but almost every function there is a singular presubmit case. So the benefit of splitting it have not justified break the pattern for me. Instead I put them in a way to make the process clearer for both case.